rgw/v2/tests/s3_swift/test_bucket_policy_with_tenant_user.py

```diff
@@ -262,6 +265,81 @@ def test_exec(config, ssh_con):
                     raise AssertionError(
                         f"Failed to perform get_bucket_location: {err}"
                     )
+            if config.test_ops.get("lifecycle_configuration", False):
```

Comment:
Here For PUT and get life cycle we can use reusable.put_bucket_lifecycle

can make little modification in put_bucket_lifecycle to achieve current requirement
lets take two extra param to the method with default value true:
1. put_lifecycle = true
2. get_lifecycle - true

and split the workflow in method:
if put_lifecycle:
   <put-lifecycle-workflow>
if get_lifecycle:
    <get-lifecycle-workflow>

Reply:
updated as per comment, and also updated the same log